blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -44,17 +52,18 @@ export default class PipelinePage extends Component {
                         <TabLink to="/pr">Pull Requests</TabLink>
                     </PageTabs>
                 </PageHeader>
-                {React.cloneElement(this.props.children, { pipeline })}
+                {this.props.children}
             </Page>
         );
     }
 }
 
 PipelinePage.propTypes = {
-    children: object,
+    children: PropTypes.any,
```

Comment:
Hm, this is definitely a code style difference of opinion. Our files are currently mixed, which is bad. Defining the vars this way into these common english terms includes `object` and `array` which are only a case differences from the Javascript `Object` and `Array` types and alone should be enough to not do it, in my opinion. On top of it, changing the prop types often requires a 2 line change, instead of a 1-line change. It will also lead to lint issues when people forget to remove unused `PropType` definitions. It's also poor code practice to have the `const` definition so far away from the usage, if that is the preferred way to do it. So, I strongly disagree that the code before is in any way 'cleaner'.